code/frontend/src/app/wo-clearing/wo-clearing.component.ts

```diff
@@ -132,45 +136,53 @@ export class WoClearingComponent implements OnInit {
         }
         this.filteredProtocols = filtered;
     }
-    // filtr for protocol 
-    private addToTable (ordersNotReady:Order[]) :void
-    {
+
+    private addToTable (ordersNotReady:Order[]) :void {
         this.ordersNotReady=[];
         this.ordersReadyForProtocol=[];
-        for (let order of ordersNotReady )
-        {      
+        for (let order of ordersNotReady ){      
             if (!this.toolsService.isReadyForProtocol(order,true))
-            {
```

Comment:
zostawmy ta zmiane, ale wg mnie warto majac nawet jedna linike if..else zostawiac nawiasy, bo jesli kiedykowiek trzeba bedzie dodac tam kod nawet cos w stylu console.log to trzeba dopisywac nawiasy i mozna sie pomylic.